util/topgen/intermodule.py

```diff
@@ -233,6 +233,10 @@ def elab_intermodule(topcfg: OrderedDict):
 
     if "ext" not in topcfg["inter_module"]:
         topcfg["inter_module"]["ext"] = []
+        topcfg["inter_signal"]["external"] = []
+
+    if "external" not in topcfg["inter_signal"]:
```

Comment:
are we going to get confused between `ext` and `external`? I've
already forgotten what `ext` stands for.

Reply:
Let's fix it to `external`, which is easier to understand (but more typing) :) Thanks for suggestion.